src/sql/src/session/vars.rs

```diff
@@ -1311,7 +1311,7 @@ static LINEAR_JOIN_YIELDING: Lazy<ServerVar<String>> = Lazy::new(|| ServerVar {
     value: &DEFAULT_LINEAR_JOIN_YIELDING,
     description:
         "The yielding behavior compute rendering should apply for linear join operators. Either \
-         'work:<amount>' or 'time:<milliseconds>'.",
+         'work:<amount>' or 'time:<milliseconds>' or 'work:<amount>,time:<milliseconds>'.",
```

Comment:
We should document the behavior of overriding values somewhere:
* The default is to yield after producing 1M records.
* Overriding the default overrides both work and time unconditionally, meaning that `time:<milliseconds>` will not yield by work. This might be surprising, but I believe it's OK since only us can set the value.

Reply:
I've extended the description of this flag.